### PR TITLE
mount: add page

### PR DESCRIPTION
**Pull request by pxgamer**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/windows/mount.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+- Mount a share to the "Z" drive letter:
+
+`mount \\{{computer_name}}\{{share_name}} {{Z:}}`
+
+- Mount a share to the next available drive letter:
+
+`mount \\{{computer_name}}\{{share_name}} *`
+
+- Mount a share with a timeout in seconds:
```
Comment (schneiderl) [inline pages/windows/mount.md]: What exactly is this timeout for?

Comment (pxgamer) [reply pages/windows/mount.md]: The connection to the share. 🤔

Comment (einverne) [reply pages/windows/mount.md]: The official document shows:

>  Sets the time-out value in seconds for a remote procedure call (RPC). Acceptable values are 0.8, 0.9, and any integer in the range 1-60; the default is 0.8.

Should we mention the value range here?

Comment (pxgamer) [reply pages/windows/mount.md]: Probably would be good. 👍 Not sure on the wording though.

Comment (sbrl) [reply pages/windows/mount.md]:
```suggestion
- Mount a share with a read timeout in seconds (defaults to 0.8, can be 0.9 or 1 to 60):
```

Comment (pxgamer) [reply pages/windows/mount.md]: Thanks @sbrl!

**Pull request merged**